utils/funcexp/func_char.cpp

```diff
@@ -154,15 +156,18 @@ string Func_char::getStrVal(Row& row,
                         tmpval++;
 
                     value = datatypes::Decimal::getInt32FromWideDecimal(tmpval);
+*/
+                    // rounding by the left over
+                    value = static_cast<int32_t>(d.getRoundedIntegralPart(roundingFactor));
```

Comment:
Same here, why not just delete the old code? Will it be needed later?

Reply:
Removed.